modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/GcalControllerEpics.scala

```diff
@@ -22,6 +22,7 @@ object GcalControllerEpics extends GcalController {
       v match {
         case BinaryOnOff.OFF => LampState.Off
         case BinaryOnOff.ON  => LampState.On
+        case _               => sys.error("Cannot happen")
```

Comment:
What is the error and which options have you enabled? There are the two that TLS adds:

```scala
"-Xstrict-patmat-analysis",      // more accurate reporting of failures of match exhaustivity
"-Xlint:strict-unsealed-patmat", // warn on inexhaustive matches against unsealed traits
```

Do you get this behavior if you turn the one or both off?

It's definitely a bug if it doesn't understand Java enums.